nixos/modules/services/web-apps/invidious.nix

```diff
@@ -155,6 +159,27 @@ in
       description = lib.mdDoc "The Invidious package to use.";
     };
 
+    automaticRestart = {
+      enable = lib.mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
Since Invidious must be restarted often, I think this option should default to `true`.

Reply:
Yes indeed. I changed the default value to `true`.